src/app/course/_services/goal.service.ts

```diff
@@ -25,6 +25,11 @@ export class GoalService {
         return this.http.get<Goal[]>(url).pipe(catchError(this.apiService.handleError<Goal[]>('Unable to fetch goals.')))
     }
 
+    getGoalSuggestions() {
```

Comment:
If the suggestions are out of scope of this PR please remove this as well.
PS. the error message should be 'Unable to fetch goal suggestions'.